Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15146,6 +15146,24 @@ void WebPageProxy::requestTargetedElement(const API::TargetedElementRequest& req
     });
 }
 
+void WebPageProxy::requestAllTargetableElements(WebCore::FloatSize minimumSize, CompletionHandler<void(const Vector<Vector<Ref<API::TargetedElementInfo>>>&)>&& completion)
+{
+    if (!hasRunningProcess())
+        return completion({ });
+
+    sendWithAsyncReply(Messages::WebPage::RequestAllTargetableElements(minimumSize), [completion = WTFMove(completion), weakThis = WeakPtr { *this }](Vector<Vector<WebCore::TargetedElementInfo>>&& elements) mutable {
+        RefPtr protectedThis = weakThis.get();
+        if (!protectedThis)
+            return completion({ });
+
+        completion(WTF::map(WTFMove(elements), [protectedThis](auto&& subelements) {
+            return WTF::map(WTFMove(subelements), [protectedThis](auto&& subelement) {
+                return API::TargetedElementInfo::create(*protectedThis, WTFMove(subelement));
```

Comment:
`std::move(auto&&)` is ill-defined since this is a forwarding reference.


```suggestion
            return WTF::map(std::forward<decltype(subelements)>(subelements), [protectedThis](auto&& subelement) {
                return API::TargetedElementInfo::create(*protectedThis, std::forward<decltype(subelement)>(subelement));
```

Reply:
In hindsight, we aren't using the forwarding reference to bind to other reference types, so a `std::move` here is fine and sufficient. Excuse the pedantry :)